cmd/local/conf/defaults_test.go

```diff
@@ -69,11 +69,14 @@ func TestSetViperDefaults(t *testing.T) {
 		{conf.KeyCollectJStack, true},
 		{conf.KeyCollectSystemTablesExport, true},
 		{conf.KeyCollectWLM, true},
+		{conf.KeyCollectTtop, true},
 		{conf.KeyCollectKVStoreReport, true},
 		{conf.KeyDremioJStackTimeSeconds, defaultCaptureSeconds},
 		{conf.KeyDremioJFRTimeSeconds, defaultCaptureSeconds},
 		{conf.KeyNodeMetricsCollectDurationSeconds, defaultCaptureSeconds},
 		{conf.KeyDremioJStackFreqSeconds, 1},
+		{conf.KeyDremioTtopFreqSeconds, 1},
+		{conf.KeyDremioTtopTimeSeconds, 30},
```

Comment:
should we keep the same default as the thread dumps (60) ?

Reply:
it is actually just shoud use the variable will fix that